### ISSUE TITLE
[CLOSED] ArgumentOutOfRangeException on HeliosTextBox

### ISSUE THREAD

**Issue by BlueFinBima** [BlueFinBima/test1#158]
<a href="https://github.com/BlueFinBima"><img src="https://avatars0.githubusercontent.com/u/18526232?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [BlueFinBima](https://github.com/BlueFinBima)**
_Tuesday Sep 10, 2019 at 19:02 GMT_
_Originally opened as https://github.com/BlueFinBima/Helios/issues/158_

----

Profile Editor, drag a TextDisplay onto a monitor and these exceptions are thrown
```
System.Windows.Data Error: 17 : Cannot get 'Item[]' value (type 'ValidationError') from '(Validation.Errors)' (type 'ReadOnlyObservableCollection`1'). BindingExpression:Path=(0)[0].ErrorContent; DataItem='HeliosTextBox' (Name=''); target element is 'HeliosTextBox' (Name=''); target property is 'ToolTip' (type 'Object') ArgumentOutOfRangeException:'System.ArgumentOutOfRangeException: Specified argument was out of the range of valid values.
Parameter name: index'
```
also

```
System.Windows.Data Error: 40 : BindingExpression path error: 'UseParserDictionary' property not found on 'object' ''TextDisplay' (HashCode=63231703)'. BindingExpression:Path=Control.UseParserDictionary; DataItem='TextDisplayAppearancePropertyEditor' (Name=''); target element is 'CheckBox' (Name=''); target property is 'IsChecked' (type 'Nullable`1')
```

**Comment by BlueFinBima** [comment]
<a href="https://github.com/BlueFinBima"><img src="https://avatars0.githubusercontent.com/u/18526232?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [BlueFinBima](https://github.com/BlueFinBima)**
_Wednesday Sep 11, 2019 at 06:45 GMT_

----

Can't get VS to stop on the ArgumentOutOfRangeException for some reason.  I'm currently thinking this is an issue with TextDisplayAppearancePropertyEditor.xaml.
Appears benign, but it would be better if these were not coming out.

**Comment by BlueFinBima** [comment]
<a href="https://github.com/derammo"><img src="https://avatars0.githubusercontent.com/u/817160?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [derammo](https://github.com/derammo)**
_Monday Jan 20, 2020 at 21:03 GMT_

----

I corrected all the errors, but not sure the tooltip actually displays?   Maybe check that part or otherwise we ignore it.   I don't have time right now to figure out if the tooltip feature actually works.

**Comment by BlueFinBima** [comment]
<a href="https://github.com/BlueFinBima"><img src="https://avatars0.githubusercontent.com/u/18526232?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [BlueFinBima](https://github.com/BlueFinBima)**
_Sunday Jan 26, 2020 at 15:40 GMT_

----

Fix is candidate for release after 1.4.2019.1005
Awesome job @derammo
